src/helm/proxy/clients/http_client.py

```diff
@@ -0,0 +1,151 @@
+from dataclasses import asdict
+
+from helm.common.cache import Cache, CacheConfig
+from helm.common.request import (
+    Request,
+    RequestResult,
+    Sequence,
+    Token,
+    EMBEDDING_UNAVAILABLE_REQUEST_RESULT,
+)
+from helm.common.tokenization_request import (
+    DecodeRequest,
+    DecodeRequestResult,
+    TokenizationRequest,
+    TokenizationRequestResult,
+    TokenizationToken,
+)
+from .client import Client, wrap_request_time
+
+import requests
+
+
+class HTTPClient(Client):
+    """Implements a simple HTTP client."""
+
+    def __init__(
+        self,
+        cache_config: CacheConfig,
+        base_url: str = "http://localhost",
+        port: int = 8080,
+        timeout: int = 10,
```

Comment:
Is this timeout a constraint for the challenge?

Reply:
@drisspg  can we increase the default timeout? 10 seconds is too small for: toy-submission but using 7B LLaMA2, on 4090, for GSM scenario. I believe that is within the specs of the competition you're running? :)